ui/sign_in_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+describe 'Sign in page' do
+  context 'when unauthenticated' do
+    include_context 'as guest'
+
+    before do
+      ui.sign_in.load
+    end
+
+    it 'Warning is displayed when the user clicks login with blank username' do
+      ui.sign_in.with_email_and_password('', '')
+      expect(ui.sign_in).to have_warning_title
```

Comment:
I think we can also assert the message being the same in both these contexts. 
`Enter a valid email address and password. Your account will be locked after 5 unsuccessful attempts. We will email you instructions to unlock your account.`